src/js/common/schemaform/FormPage.jsx

```diff
@@ -53,6 +53,14 @@ class FormPage extends React.Component {
     focusForm();
   }
 
+  // A successful form save will mean we go from pending to success,
+  // and we need to redirect
+  componentWillReceiveProps(newProps) {
+    if (this.props.form.savedStatus === SAVE_STATUSES.pending && newProps.form.savedStatus === SAVE_STATUSES.success) {
+      this.props.router.push(`${newProps.urlPrefix || ''}form-saved`);
```

Comment:
Actually, it does! Just a `/`. I had been assuming it's not required to have one, I don't want to change that now. Having to put `/` for hca seems hackish.